src/FrequencyBalancerAgent.cpp

```diff
@@ -0,0 +1,644 @@
+/*
+ * Copyright (c) 2015 - 2023, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "config.h"
+
+#include "FrequencyBalancerAgent.hpp"
+
+#include <unistd.h>
+
+#include "Environment.hpp"
+#include "FrequencyGovernor.hpp"
+#include "FrequencyLimitDetector.hpp"
+#include "FrequencyTimeBalancer.hpp"
+#include "PowerGovernor.hpp"
+#include "SSTClosGovernor.hpp"
+#include "Waiter.hpp"
+#include "geopm/Exception.hpp"
+#include "geopm/Helper.hpp"
+#include "geopm/PlatformIO.hpp"
+#include "geopm/PlatformTopo.hpp"
+#include "geopm_debug.hpp"
+#include "geopm_hash.h"
+#include "geopm_hint.h"
+#include "geopm_time.h"
+#include "geopm_topo.h"
+
+#include <algorithm>
+#include <chrono>
+#include <cmath>
+#include <exception>
+#include <iterator>
+#include <numeric>
+#include <thread>
+#include <utility>
```

Comment:
```suggestion
#include <unistd.h>
#include <algorithm>
#include <chrono>
#include <cmath>
#include <exception>
#include <iterator>
#include <numeric>
#include <thread>
#include <utility>

#include "geopm/Exception.hpp"
#include "geopm/Helper.hpp"
#include "geopm/PlatformIO.hpp"
#include "geopm/PlatformTopo.hpp"
#include "geopm_debug.hpp" // What's this for?

#include "geopm_hash.h"
#include "geopm_hint.h"
#include "geopm_time.h"
#include "geopm_topo.h"
#include "Environment.hpp"
#include "FrequencyGovernor.hpp"
#include "FrequencyLimitDetector.hpp"
#include "FrequencyTimeBalancer.hpp"
#include "PowerGovernor.hpp"
#include "SSTClosGovernor.hpp"
#include "Waiter.hpp"
```
Check out service/src/PlatformIO.cpp for the desired ordering, but the gist is:

1. config.h
2. ClassImp.hpp
3. std C headers.
4. std C++ headers.
5. Public geopm headers.
6. Private geopm c headers.
7. Private geopm c++ headers.